tools/idllib/file_write_string.pro

```diff
@@ -0,0 +1,36 @@
+; Write a string to a file as an attribute
+FUNCTION file_write_string, handle, varname, data
+
+  CATCH, errcode
+  
+  IF errcode NE 0 THEN BEGIN
+    ; error occurred
+    PRINT, "Error occurred in file_write: "+!ERROR_STATE.MSG
+    PRINT, "=> Couldn't write variable '"+varname+"'"
+    RETURN, 1
+  ENDIF
+
+  IF handle.type EQ 1 THEN BEGIN
```

Comment:
I don't think we support PDB anymore, though this might be useful for older versions of BOUT(++)

Reply:
I haven't checked if the PDB branch works. The function was copied from `file_write.pro`, which had PDB support, so I just left it in.